pkg/cgroups/systemd.go

```diff
@@ -1,3 +1,6 @@
+//go:build !linux
+// +build !linux
```

Comment:
@cdoern can you shed some light on what's the purpose of having this systemd-specific functionality (`systemdCreate`, `systemdDestroyConn`) for non-linux platforms? Perhaps I am missing something but I can't see a way to ever use this code.